src/Package_Command.php

```diff
@@ -202,6 +202,12 @@ public function install( $args, $assoc_args ) {
 			preg_match( '#([^:\/]+\/[^\/]+)\.git#', $package_name, $matches );
 			if ( ! empty( $matches[1] ) ) {
 				$package_name = $matches[1];
+				$raw_content_url = 'https://raw.githubusercontent.com/' . $package_name . '/master/composer.json';
+				$composer_content_as_array = json_decode( $this->get_data_from_url( $raw_content_url ), true );
```

Comment:
Why not use `WP_CLI\Utils\http_request()`?

Reply:
My bad, I was unaware of this function :sweat: I'll make the changes